### ISSUE TITLE
Create the art that will be used in the project

### ISSUE THREAD

**Issue by samihenrique** [vitta-health/attiv#38]

**Comment by gambardellawill** [comment]
It's still a draft, but I want to suggest this logo I drew as the new logo for attiv.

![logo_attiv2 svg](https://user-images.githubusercontent.com/6922944/66959044-f5442300-f03f-11e9-81b9-d4fd631ddc20.png)

**Comment by taciomedeiros** [comment]
When i think about the logo i prefer something cleaner like laravel or just a font type like express

**Comment by gambardellawill** [comment]
Another sketch for the brand.

![attiv_text](https://user-images.githubusercontent.com/6922944/66962109-9930cd00-f046-11e9-9251-63fd9359e63d.png)

**Comment by gambardellawill** [comment]
Another sketch for the brand.

![attiv_text](https://user-images.githubusercontent.com/6922944/66962314-13f9e800-f047-11e9-905f-294f692764b2.png)

**Comment by taciomedeiros** [comment]
@gambardellawill  i like the first one! But i think it's best shape will be without the initial point.

@samihenrique  @rlrezende  @marcioibm  @filipeforattini ?

**Comment by gambardellawill** [comment]
@taciomedeiros I had initially thought about replacing the dot with curly braces on both sides of the attiv logo, as curly braces lend well to the concept of attiv being a Javascript framework of sorts.

**Comment by samihenrique** [comment]
> @gambardellawill i like the first one! But i think it's best shape will be without the initial point.
> 
> @samihenrique @rlrezende @marcioibm @filipeforattini ?

The three were excellent, I would choose the one with dark background.

I really liked the use of the dot at the beginning, because it resembles a configuration file.

We could make several versions of the logo being some colored to make it more youthful.

**Comment by luizzzdev** [comment]
The three are awesome!

**Comment by marcioibm** [comment]
I don't think we need to pick a color right now! It's good to be able to choose which one works better in each situation! I really liked the logos!!
